site/src/pages/TemplatesPage/TemplatesPageView.tsx

```diff
@@ -38,12 +39,17 @@ import {
 } from "../../components/Tooltips/HelpTooltip/HelpTooltip"
 
 export const Language = {
+  buildTime: (buildTimeSec: number): string =>
+    buildTimeSec === -1
+      ? "Unknown"
+      : dayjs.duration(buildTimeSec, "seconds").humanize(),
   developerCount: (activeCount: number): string => {
     return `${formatTemplateActiveDevelopers(activeCount)} developer${
       activeCount !== 1 ? "s" : ""
     }`
   },
   nameLabel: "Name",
+  buildTimeLabel: "Build time",
```

Comment:
We'll want to move this in to a translation file - `site/src/i18n/en/templatesPage.json` will work fine.
You can look at[ this explanatory PR](https://github.com/coder/coder/pull/3682) to see how how use translations in the component file.

Reply:
Where are these strings actually being used?

Reply:
I think if we want to use translations across the board we should migrate everything in one fell swoop.

Reply:
I'm happy to handle that; maybe toward the end of next week.

Reply:
Up to you. I have no preference, except for consistency.